Makefile

```diff
@@ -10,6 +10,11 @@ vocab_limit=
 vocab_str=10k
 
 
+ukwiki_vectors: \
+        datasets/ukwiki-$(dump_date)-plaintext.w_labels.txt \
+        datasets/ukwiki-$(dump_date)-learned_vectors.$(vector_dimensions)_cell.vec.bz2 \
+        datasets/ukwiki-$(dump_date)-learned_vectors.$(vector_dimensions)_cell.$(vocab_str).kv
```

Comment:
You only need line 16 here.  Line 14 and 15 are implied when you have line 16.

Reply:
Oh. Waoh. Does this mean line 16 will re-run 14 and 15 or just use the data there?

Reply:
It'll just re-use the data there.  Essential what this rule is saying is:

```
In order to satisfy 'ukwiki_vectors', we need to solve all dependencies for the plaintext, .vec, and .kv files.
```

When I think you mean to say:

```
In order to satisfy 'ukwiki_vectors', we need to solve all dependencies for the .kv file (which implicitly include the plaintext and .vec files).
```